admin/controllers/check-started.js

```diff
@@ -0,0 +1,33 @@
+const jwtService = require('../services/jwt.service')
+const pinService = require('../services/pin.service')
+const apiResponse = require('./api-response')
+
+/**
+ * Expires the pupil's pin
+ * @param req
+ * @param res
+ * @returns { object }
+ */
+
+const preCheck = async (req, res) => {
```

Comment:
can we call this checkStarted or something similar?  preCheck doesnt feel accurate.

Reply:
@GuyHarwood changed